README_Instructions.md

```diff
@@ -0,0 +1,32 @@
+# EdgeX-TAF execution in Pipeline
```

Comment:
We use data.txt throughout these files.  I think one script populates that data and shares it (or something like that).  Could we add a comment for that somewhere in this MD file describing that workflow?

Reply:
Okay